example/test_calendar.yaml

```diff
@@ -1,12 +1,15 @@
+meta:
+  tz: America/Los_Angeles
+
 events:
   - summary: Event of the Century
-    begin: 2021-09-21 15:00:00 -07:00
```

Comment:
This now removes the only examples we had with a timezone in an entry.  We'd also want to check whether timezones are still correctly interpreted, even if the meta tz is specified.

Reply:
I moved the explicit timezone to the event below (which previously had no timezone), so that the first example is (what I think would be) the best practice, and after that is the other example.

Though, now there is not an example of a floating event that has no timezone at all.  Perhaps that can be ignored for now, I guess floating events are probably not a main target of this.